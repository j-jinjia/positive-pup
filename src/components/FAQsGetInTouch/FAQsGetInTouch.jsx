import Button from "../Button/Button";
import Header from "../Header/Header";
import "./FAQsGetInTouch.scss";
import { Link } from "react-router-dom";

const FAQsGetInTouch = () => {
  return (
    <div className="faq-get-in-touch">
      <Header
        className="faq-get-in-touch__heading"
        subheadingText="Can't find an answer to your Question?"
      />
      <p className="faq-get-in-touch__paragraph">
        Please get in touch below if you can’t find an answer you’re looking
        for. I’d love to hear from you! Thanks Mel
      </p>

      <Button
        buttonText={"GET IN TOUCH"}
        isSecondary
        link={<Link to="/get-in-touch">GET IN TOUCH</Link>}
      />
    </div>
  );
};

export default FAQsGetInTouch;
