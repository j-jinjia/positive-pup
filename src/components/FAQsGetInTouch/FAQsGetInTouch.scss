@use "../../styles/variables" as *;
* {
  margin: 0;
  padding: 0;
}

.faq-get-in-touch {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;

  &__heading {
    font-size: $font-size-medium;
    font-weight: 600;
    font-family: $font-header;
    margin-bottom: 0;
  }

  &__paragraph {
    text-align: center;
    font-size: $font-size-default;
    font-family: $font-general;
    line-height: 150%;
    max-width: 36.375rem;
    align-items: center;
    margin-bottom: 2.5rem;
  }

  .button {
    font-family: $font-general;
    color: $colour-primary;
    font-weight: 700;
    font-size: $font-size-default;
    margin-bottom: 6.25rem;
  }
}
