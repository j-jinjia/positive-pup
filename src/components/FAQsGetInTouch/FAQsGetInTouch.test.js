import { customRender } from "../../utils/testUtils";
import FAQsGetInTouch from "./FAQsGetInTouch";

it("should match the FAQS Get in Touch component snapshot", () => {
  const { container } = customRender(<FAQsGetInTouch />);

  expect(container).toMatchSnapshot();
});
