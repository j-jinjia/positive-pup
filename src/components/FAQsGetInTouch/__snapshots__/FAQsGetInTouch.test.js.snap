// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the FAQS Get in Touch component snapshot 1`] = `
<div>
  <div
    class="faq-get-in-touch"
  >
    <div
      class="header"
    >
      <h2
        class="header__subheading"
      >
        Can't find an answer to your Question?
      </h2>
    </div>
    <p
      class="faq-get-in-touch__paragraph"
    >
      Please get in touch below if you can’t find an answer you’re looking for. I’d love to hear from you! Thanks Mel
    </p>
    <div
      class="button button--secondary button--link"
    >
      <a
        href="/get-in-touch"
      >
        GET IN TOUCH
      </a>
    </div>
  </div>
</div>
`;
